cmake/Modules/Findblis.cmake

```diff
@@ -0,0 +1,42 @@
+SET(BLIS_INCLUDE_SEARCH_PATHS
```

Comment:
This file is necessary since it defines BLIS_INCLUDE_DIR and BLIS_LIB which are needed in cmake/Dependencies.cmake:
elseif(BLAS STREQUAL "blis")
  find_package(blis REQUIRED)
  include_directories(SYSTEM ${BLIS_INCLUDE_DIR})
  list(APPEND Caffe2_PUBLIC_DEPENDENCY_LIBS ${BLIS_LIB})

Without this it will not be able to find  blis include and blis library paths and it will throw the same error as above (which we tried with FLAME option)

Reply:
Why not use the FindBLAS functions? BLIS is a BLAS and hence should provide header and library through it?